torch/testing/_internal/common_methods_invocations.py

```diff
@@ -358,6 +358,28 @@ def sample_inputs_tensor_split(op_info, device, dtype, requires_grad):
                         args=(torch.tensor([1, 2, 3]),),
                         kwargs=dict(dim=1)),)
 
+def sample_inputs_linalg_matrix_power(op_info, device, dtype, requires_grad):
+    # (<matrix_size>, (<batch_sizes, ...>))
+    test_sizes = [
+        (1, ()),
+        (S, ()),
```

Comment:
Let's use 2 instead of S

Reply:
Ok, test duration came down to 22 seconds with this change

Reply:
That should be (barely) OK for now.